app/CMakeLists.txt

```diff
@@ -356,6 +356,13 @@ if(CELERITAS_BUILD_DEMOS AND CELERITAS_USE_Geant4)
     HepMC3::HepMC3
   )
   if(CELERITAS_BUILD_TESTS)
+    set(DEMO_MAX_NUM_TRACKS 1024)
```

Comment:
It's inserted into the macro file at configuration time so that our demo parameters change based on the configuration (GPU or not). It was the only way I could think of to do that without copy pasting the file; do you have an alternative way to suggest?

Reply:
Okay, overlooked the part for CELERITAS_USE_CUDA.  No better idea and good for now.  Thanks.

Reply:
I clarified the CMake code a bit in one of the follow-on pull requests (so it's an `if/else` rather than an `if` overriding it).